docs/setup/kubeedge_install_keadm.md

```diff
@@ -0,0 +1,219 @@
+# Setup from KubeEdge Installer
+
+Please refer to KubeEdge Installer proposal document for details on the motivation of having KubeEdge Installer.
+It also explains the functionality of the proposed commands.
+[KubeEdge Installer Doc](https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/keadm-scope.md/)
+
+## Limitation
+
+- Currently support of `KubeEdge installer` is available only for Ubuntu OS. CentOS support is in-progress.
```

Comment:
Does it support all versions of Ubuntu OS? If not then please list specific versions that are supported.

Reply:
@anvithks Currently, we do not maintain which version of Ubuntu it supports. Ideally, it should be working on every version of Ubuntu OS.